kubernetes/.swagger-codegen/VERSION

```diff
@@ -0,0 +1 @@
+2.3.0-SNAPSHOT
```

Comment:
To supply a version in the pom.xml that depends on the git commit we'd have to change how swagger-codegen is built. The version specified here is resolved by maven, not our build script, unfortunately. If our automation did this, it might get messy and interfere with users local development environments since to pull a -SNAPSHOT release your machines local maven repo is used.

It might make sense (for now in documentation only) require the latest @ HEAD version of swagger-codegen to be checked out & built locally each time the client generation is run (until 2.3.0 is released). This should technically agree with semver, given that the maintainers of swagger-codegen are diligent in segmenting changes to each release branch by feature/bug-fix/breaking-change/etc...